src/main/java/io/lettuce/core/Transports.java

```diff
@@ -34,7 +34,7 @@
  * @author Yohei Ueki
  * @since 4.4
  */
-class Transports {
+public class Transports {
```

Comment:
[Supplemental Comments]
I made this public because this is now referenced by outside of the package.
https://github.com/lettuce-io/lettuce-core/pull/1517/commits/ec85577c80e82ff8a48c87535aa1c8433a92f1a0#diff-724688b7234bd44bd8e5f08ce8b0a616ecaf11729ebd99ad72d510431878a6f4R60-R63

Reply:
We should move `Transports` to the `resource` package, that's something we can do during the merge.